projects/module-2/whats-cookin-part-two-agile.md

```diff
@@ -0,0 +1,212 @@
+
+## Daily Check-Ins
+
+Every work day, your group should do a check-in as a whole group, preferably live over Zoom, and discuss at least these things:
+
+- What is the state of the project so far? (update your GitHub project board if necessary)
+- What is each person working on today?
+- Are there any blockers, and what is your plan to get through the blocker?
+
+Your daily check-in schedule (sometimes called “stand-ups”) should be outline in your DTR so the whole group is aware of the meeting.
+
+## Retrospective
+
+A retrospective (retro) is a look back at the recent development period, in this case, Part 1 of the project, in order to improve future development **process** and **workflow** as a team. For your retrospective, meet with your team to discuss the following questions:
+
+- What helped you to be successful as a team?
+- What caused the problems that you had in Part 1? For example: When we got stuck, how long did we grind before reaching out for help?
+- Were daily check-ins productive and helpful? If not, what would you change?
+- What questions do we have about Part 2 of the project?
+- Is there anything we should change in our DTR?
+
+When you’ve finished your retro, DM your project manager two things from your retrospective (see below) that you want to keep doing and two things that you want to improve on as a group heading into Part 2.
+
+## Applying Instructor Feedback from Part 1
+
+- Implement instructor feedback from Part 1. Be sure to ask your instructor about any clarifying questions you have about feedback.
+- Consider any additional refactoring opportunities:
+    - Identify redundant code in your classes and opportunities for DRYing it up
+    - Refactor **within** your classes to create dynamic methods that use arguments/parameters for changing their behavior.
+
+## Functionality
+
+You must complete all of the remaining user stories from the [What’s Cookin Part 1 Spec](https://frontend.turing.edu/projects/whats-cookin-part-one.html). If you did not finish parts of the original requirements, this is your chance to revisit and finish all of the functionality. In addition to the Part 1 requirements you must also ***implement your instructor’s feedback*** and add ***“Iteration Four” and “Iteration Five”***.
+
+## Iteration 4 - POST
+
+**Goal**: Use a POST request to save the recipes that a user wants to cook (favorite) to the local API. That way when the page is refreshed, the user can still see their favorited recipe(s). The documentation for the API is outlined below.
+
+Being able to do a DELETE request is not a requirement for this project. So to delete a user’s favorite recipes, you can reset the data on the local server by restarting the local server on your machine. Do not worry about “un-favoriting” a recipe via a DELETE request; restart the API server instead.
```

Comment:
For line 66, I could see students being confused on whether this is telling them to manually restart the local server vs writing code that will do that on click of a button, etc.   Can we clarify that in the spec?